Content revision for blockchain bridges (#11479)
ethereum/ethereum-org-website: src/content/bridges/index.md

# Blockchain bridges {#prerequisites}

> _Web3 has evolved into an ecosystem of L1 blockchains and L2 scaling solutions, each designed with unique capabilities and trade-offs. As the number of blockchain protocols increases, so does [the demand to move assets across chains](<https://dune.xyz/eliasimos/Bridge-Away-(from-Ethereum)>). To fulfill this demand, we need bridges._

Reviewer: 👍

Let's consider an example:

> You're from the USA and are planning a trip to Europe, where only EUR is accepted. You can use a currency exchange for a small fee to exchange your USD for EUR.

Reviewer: I'm not sure if there is value-add here.. I feel like the simplicity of the three short sentences was intentional here. Example:>Setup.>Problem.>Solution. Would probably leave this

```suggestion
You're from the USA and are planning a trip to Europe. You have USD, but you need EUR to spend. To exchange your USD for EUR you can use a currency exchange for a small fee.
```

### Own native crypto assets {#own-native}

> Let’s say you want to own native Bitcoin (BTC), but you only have funds on Ethereum Mainnet. To gain exposure to BTC on Ethereum, you can buy Wrapped Bitcoin (WBTC). However, WBTC is an ERC-20 token native to the Ethereum network, which means it’s an Ethereum version of Bitcoin and not the original asset on the Bitcoin blockchain. To own a native BTC, you would have to bridge your assets from Ethereum to Bitcoin using a bridge. This will bridge your WBTC and convert it into a native BTC. Alternatively, you might own BTC and want to use it in Ethereum DeFi protocols. This would require bridging the other way, from BTC to WBTC which can then be used as an asset on Ethereum.

Reviewer: BTC is being used in the plural form, not singular; "some" would be more appropriate here, but personally not sure the change is needed

Reviewer: ```suggestion
Let’s say you want to own native Bitcoin (BTC), but you only have funds on Ethereum Mainnet. To gain exposure to BTC on Ethereum, you can buy Wrapped Bitcoin (WBTC). However, WBTC is an ERC-20 token native to the Ethereum network, which means it’s an Ethereum version of Bitcoin and not the original asset on the Bitcoin blockchain. To own native BTC, you would have to bridge your assets from Ethereum to Bitcoin using a bridge. This will bridge your WBTC and convert it into a native BTC. Alternatively, you might own BTC and want to use it in Ethereum DeFi protocols. This would require bridging the other way, from BTC to WBTC which can then be used as an asset on Ethereum.
```

1. Manual Checkpoints — operated by officials who manually check all the details of your ticket and identity before handing over the boarding pass.
2. Self Check-In — operated by a machine where you put in your flight details and receive the boarding pass if everything checks out.

> A manual checkpoint is similar to a trusted model as it depends upon a third party, i.e., the officials, for its operations. As a user, you trust the officials to make the right decisions and use your private information correctly.

Reviewer: 👍
